frontend/src/pages/TracesExplorer/TracesExplorer.styles.scss

```diff
@@ -49,7 +49,7 @@
 	}
 
 	.trace-explorer {
-		width: 100%;
+		width: calc(100% - 260px);
```

Comment:
@ahmadshaheer : Why -260px?

Reply:
the traces [filters](https://github.com/SigNoz/signoz/blob/a50bd9912aa04651a513ef43961a43e090d28ec6/frontend/src/pages/TracesExplorer/TracesExplorer.styles.scss#L38-L39) have a width of 260px. I subtracted it from 100% to ensure that the traces explorer slide to the side when there are too many columns.